### ISSUE TITLE
Implement back button to previous page

### ISSUE THREAD

**Issue by i315315** [i315315/amos2022ws02-automotive-test-app#19]
## User story
1. As a User
2. I want to be able to leave an area in the test app where all use cases for navigation are collected
3. So that I can easily switch back to the previous page.

I would like to be able to get back to the previous page with a "Back" button.
The "back"-button should be implemented in a way that it can be reused for other screens as well.

## Acceptance criteria
* In the test app there is a possibility to go back to the previous page.
* Tapping the "Back" button takes you back to the previous page (from which the navigation context was entered).
* The button can be reused for other screens as well.

## Notes
Usually, Android Automotive should have a functionality like this already implemented.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
